collect_app/src/test/java/org/odk/collect/android/formmanagement/FormSourceProviderTest.kt

```diff
@@ -1,27 +0,0 @@
-package org.odk.collect.android.formmanagement
-
-import androidx.test.espresso.matcher.ViewMatchers.assertThat
-import org.hamcrest.Matchers.`is`
-import org.junit.Test
-import org.mockito.kotlin.mock
-import org.odk.collect.android.openrosa.OpenRosaFormSource
-import org.odk.collect.settings.keys.ProjectKeys
-import org.odk.collect.shared.settings.InMemSettings
-
-class FormSourceProviderTest {
```

Comment:
I got rid of this because it meant keeping `getWebCredentialsUtils` and I confirmed that removing the logic to load url/credentials from settings was covered by other tests now (`GetAndSubmitFormTest` and `GetBlankFormsTest` for example).